examples/hpo/gp_mpc/config_overrides/cartpole/cartpole_stab.yaml

```diff
@@ -0,0 +1,67 @@
+task_config:
```

Comment:
Why is the file structure of this example `algo/config_override/env`? All of these files are configs rt, so the directory structure should be `config_overrides/algo/env` or something like that, with `config_override` at the top level.

Reply:
Sounds good